pkg/idtools/idtools_unix.go

```diff
@@ -46,6 +46,9 @@ func mkdirAs(path string, mode os.FileMode, ownerUID, ownerGID int, mkAll, chown
 		// walk back to "/" looking for directories which do not exist
 		// and add them to the paths array for chown after creation
 		dirPath := path
+		if !strings.HasPrefix(dirPath, "/") {
+			return fmt.Errorf("path: %s should start with /", dirPath)
```

Comment:
Shouldnt we do `filepath.IsAbs` ?

Reply:
Thanks for your suggestion, I replaced with `filepath.IsAbs`